tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -29,19 +39,79 @@ public void process(final Socket connection) {
+            List<String> requestLines = getInput(reader);
+
+            HttpRequest request = new HttpRequest(requestLines);
 
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            if (request.getMethod().equals("GET") && request.getPath().equals("/login")) {
+                UserService.checkUser(
+                        request.getQueryParameter("account"),
+                        request.getQueryParameter("password")
+                );
+            }
 
-            outputStream.write(response.getBytes());
-            outputStream.flush();
-        } catch (IOException | UncheckedServletException e) {
+            String contentType = parseContentType(request.getPath());
+            String body = getResponseBody(request.getPath());
+            response(body, contentType, outputStream);
+        } catch (IOException | UncheckedServletException | URISyntaxException e) {
             log.error(e.getMessage(), e);
         }
     }
+
+    private List<String> getInput(BufferedReader reader) throws IOException {
+        List<String> lines = new ArrayList<>();
+        String line;
+        while ((line = reader.readLine()) != null && !line.isEmpty()) {
+            lines.add(line);
+        }
+        return lines;
+    }
+
+    private String parseContentType(String uri) {
+        if (uri.endsWith(".css")) {
+            return "text/css;charset=utf-8";
+        } else if (uri.endsWith(".js")) {
+            return "application/javascript;charset=utf-8";
+        } else {
+            return "text/html;charset=utf-8";
+        }
+    }
+
+    private String getResponseBody(String path) throws IOException, URISyntaxException {
+        URL resource = getResource(path);
+        if (resource == null || Files.isDirectory(Path.of(resource.toURI()))) {
+            return "Hello world!";
+        }
+        return new String(Files.readAllBytes(new File(resource.getFile()).toPath()));
+    }
+
+    private URL getResource(String path) {
+        StringBuilder targetPath = new StringBuilder("static");
+
+        if (path.equals("/login")) {
+            targetPath.append("/login.html");
+        } else {
+            targetPath.append(path);
+        }
+
+        return getClass()
+                .getClassLoader()
+                .getResource(targetPath.toString());
+    }
+
+    private void response(String body, String contentType, OutputStream outputStream) throws IOException {
+        final var response = createHttpResponse(body, contentType);
+        outputStream.write(response.getBytes());
+        outputStream.flush();
+    }
+
+    private String createHttpResponse(String body, String contentType) throws IOException {
```

Comment:
해당 부분은 없어도 무방한 부분 같아요!
```suggestion
    private String createHttpResponse(String body, String contentType) {
```

Reply:
헉 꼼꼼한 리뷰 감사해요! 없어도 되겠네요 ㅎㅎ